test/integration/IOS/ElementTest.cs

```diff
@@ -35,20 +35,23 @@ public void FindByAccessibilityIdTest()
             By byAccessibilityId = new ByAccessibilityId("ComputeSumButton");
             Assert.Multiple(() =>
             {
-                Assert.That(Is.Not.EqualTo(_driver.FindElements(MobileBy.ClassName("UIAWindow"))[1].FindElement(byAccessibilityId).Text), null);
-                Assert.That(_driver.FindElements(MobileBy.ClassName("UIAWindow"))[1].FindElements(byAccessibilityId), Is.Not.Empty);
+                Assert.That(_driver.FindElement(MobileBy.ClassName("UIAWindow")).FindElement(byAccessibilityId).Text, Is.Not.Null);
```

Comment:
I assume findElement should throw an exception if nothing is found

Reply:
It does

Reply:
Then it would be never equal to null. The result is either an element or an exception thrown

Reply:
The Assert is done on the text of the element.
So if there is no text it will be empty, and test will fail

Reply:
we may also make the check more strict and verify the text is equal to some predefined value. But the current check is also ok then.

Reply:
now we check the exact text value